client/my-sites/email/email-different-domain-owner-message/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { useTranslate } from 'i18n-calypso';
+import TrackComponentView from 'calypso/lib/analytics/track-component-view';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+
+type EmailDifferentDomainOwnerMessageProps = {
+	source: string;
+};
+
+export const EmailDifferentDomainOwnerMessage = (
+	props: EmailDifferentDomainOwnerMessageProps
+) => {
+	const { source } = props;
+
+	const translate = useTranslate();
+
+	const onClickLink = ( eventType: 'login' | 'support' ) => {
+		const properties = {
+			action: eventType,
+			source,
+		};
+
+		recordTracksEvent( `calypso_email_providers_nonowner_click`, properties );
+	};
+
+	const translateOptions = {
+		components: {
+			contactSupportLink: (
+				<a
+					href="https://wordpress.com/help/contact"
+					onClick={ () => onClickLink( 'support' ) }
+					rel="noopener noreferrer"
+					target="_blank"
+				/>
+			),
+		},
+	};
+
+	return (
+		<>
+			<TrackComponentView
+				eventName="calypso_email_providers_nonowner_impression"
+				eventProperties={ { source } }
+			/>
+			<p className="email-non-owner-message__non-owner-message">
```

Comment:
I'm not a big fan of reusing the class name of another component. Could we find a way to share styles between those two components in a cleaner way?

Reply:
It seems that it's type mistake, since we have styles for className `email-non-domain-owner-message__non-owner-message`